README.md

```diff
@@ -26,8 +26,6 @@ This project checks the [maven repository] once a day for a new version and will
 
 ## Installation
 
-You must have the `java` binary executable available on your PATH for this to work. (JDK 11 is the minimal version supported. To install OpenJDK, please visit <https://adoptium.net/>)
```

Comment:
my take is still to inform the user about dependency on JDK and how to install it if they want to install a different version (latest 23 vs 11)

maybe just mark this as optional given that it's taken care by jdk4py

Reply:
jdk4py has limitations for OS and CPU archies. (reference: https://github.com/OpenAPITools/openapi-generator-pip/issues/13#issuecomment-2514472533)

So I will change this PR to keep jdk4py in the optional deps.

Reply:
👌 

saw that comments from you earlier.

automatically installing jdk is something nice to have but not a must.

users can always use docker if they can't install jdk for whatever reason (e.g. lack admin rights)